tests/test_diagnosis_routes.py

```
@@ -0,0 +1,142 @@
+import sys
+import os
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+
+import pytest
+from httpx import AsyncClient, ASGITransport
+from unittest.mock import AsyncMock, MagicMock
```

Ruff (0.11.9)

`unittest.mock.AsyncMock` imported but unused

Remove unused import: `unittest.mock.AsyncMock`

(F401)